build-logic/packaging/src/main/kotlin/gradlebuild.public-api-jar.gradle.kts

```diff
@@ -65,12 +65,12 @@ dependencies {
         attributes.attribute(filteredAttribute, Filtering.ALL)
     }
 
-    // Filters out classes that are not exposed by our API.
-    // TODO Use actual filtering not copying
-    registerTransform(CopyPublicApiClassesTransform::class.java) {
+    // Filter and shrink the published API.
+    registerTransform(ShrinkPublicApiClassesTransform::class.java) {
         from.attribute(filteredAttribute, Filtering.ALL)
             .attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements::class.java, LibraryElements.JAR))
         to.attribute(filteredAttribute, Filtering.PUBLIC_API)
+        parameters { publicApiPackages.set(setOf("org.gradle")) }
```

Comment:
Do we have _non_ org.gradle packages on the classpath? We are filtering below to only include project dependencies

Reply:
We had some stuff in `org.gradleinternal`, but that seems like an accident. Fix here: https://github.com/gradle/gradle/pull/29591

So we shouldn't need to actually filter anything. Lemme remove the filtering then.